hickle/loaders/load_scipy.py

```diff
@@ -3,109 +3,120 @@
 import dill as pickle
 import numpy as np
 import scipy
+import copy
 from scipy import sparse
 
 # hickle imports
-from hickle.helpers import get_type_and_data
+from hickle.helpers import PyContainer,H5NodeFilterProxy
+from hickle.loaders.load_numpy import load_ndarray_dataset,create_np_array_dataset
 
 
 # %% FUNCTION DEFINITIONS
-def return_first(x):
+def return_first(x): # pragma: nocover
```

Comment:
This should be covered.

Reply:
Its a dummy which represents object_type of  h5py.groups encoding sparse matrix data in Hickle 4.x files. It is used in the SparseMatrixContainer __init__function to switch filter generator to restructure subitems to be properly loaded by hickle 5. The function itself is never called and used. So it did not make much sense to write a test for a dummy function used in loading legacy 4.x file. I will add a dummy tests for this dummy function.